src/Compilers/CSharp/Portable/Symbols/ParameterSymbol.cs

```diff
@@ -385,6 +385,22 @@ internal sealed override ObsoleteAttributeData ObsoleteAttributeData
 
         internal abstract bool IsCallerMemberName { get; }
 
+        internal abstract AttributeAnnotations FlowAnalysisAnnotations { get; }
+
+        protected (bool memberHasExtra, AttributeAnnotations annotations) TryGetExtraAttributeAnnotations()
```

Comment:
>memberHasExtra [](start = 24, length = 14)

The value of ```AttributeAnnotations``` member is not obvious to me.

Reply:
Perhaps a clear comment might help

---
In reply to: [187939172](https://github.com/dotnet/roslyn/pull/26656#discussion_r187939172) [](ancestors = 187939172)